backend/tracim_backend/tests/functional/test_favorites.py

```diff
@@ -0,0 +1,210 @@
+from http import HTTPStatus
+
+import pytest
+import transaction
+
+from tracim_backend import ContentNotFound
+from tracim_backend.app_models.contents import HTML_DOCUMENTS_TYPE
+from tracim_backend.error import ErrorCode
+from tracim_backend.lib.core.content import ContentApi
+from tracim_backend.tests.fixtures import *  # noqa: F403,F40
+
+
+@pytest.mark.usefixtures("base_fixture")
```

Comment:
I would be in favor of tests which define some test data through fixtures to avoid copy/paste of the test setting up.

Reply:
Not sure if you comment the use of base_fixture (which just add the admin) or comment the repetition of thread creation in test code. anyway I did some refactoring improvement.

Reply:
This was about the repetition of thread creation.